packages/tools/installer/scripts/installer

```diff
@@ -262,27 +262,26 @@ do_install_quick() {
         sync
 
         # configuring bootloader
+        mkdir -p $TMPDIR/part1/EFI/BOOT
         msg_progress_install "80" "Setup bootloader with boot label = $DISKLABEL_SYSTEM and disk label = $DISKLABEL_STORAGE"
         echo "DEFAULT linux" > $TMPDIR/part1/extlinux.conf
         echo "PROMPT 0" >> $TMPDIR/part1/extlinux.conf
         echo " " >> $TMPDIR/part1/extlinux.conf
         echo "LABEL linux" >> $TMPDIR/part1/extlinux.conf
         echo " KERNEL /KERNEL" >> $TMPDIR/part1/extlinux.conf
         echo " APPEND boot=LABEL=$DISKLABEL_SYSTEM disk=LABEL=$DISKLABEL_STORAGE $EXTLINUX_PARAMETERS quiet" >> $TMPDIR/part1/extlinux.conf
-        echo "set timeout=\"0\"" > $TMPDIR/part1/grub.cfg
-        echo "set default=\"LibreELEC\"" >> $TMPDIR/part1/grub.cfg
-        echo "" >> $TMPDIR/part1/grub.cfg
-        echo "menuentry \"LibreELEC\" {" >> $TMPDIR/part1/grub.cfg
-        echo "	search --set -f /KERNEL" >> $TMPDIR/part1/grub.cfg
-        echo "	linux /KERNEL boot=LABEL=$DISKLABEL_SYSTEM disk=LABEL=$DISKLABEL_STORAGE quiet" >> $TMPDIR/part1/grub.cfg
-        echo "}" >> $TMPDIR/part1/grub.cfg
+        echo "set timeout=\"0\"" > $TMPDIR/part1/EFI/BOOT/grub.cfg
```

Comment:
You could use here multiline string using `EOF` block as you did on other places. The code is much more readable.

Reply:
I did - next commit. :)

Reply:
Ah, there were to many changes to see that :)